### PR TITLE
Tao model & migration

### PR DESCRIPTION
**Pull request by NgocAnhDT**
## Related Tickets:
- [#46195](https://edu-redmine.sun-asterisk.vn/projects/hn_oe49-shoe-store/issues/46195)

## WHAT this PR does?
- Create Model & Migration

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by NgocAnhDT** [comment]
ready

**Comment by NgocAnhDT** [comment]
ready

**Comment by NgocAnhDT** [comment]
ready

File: app/Models/User.php [hunk 1]
```diff
@@ -7,11 +7,12 @@
 use Illuminate\Foundation\Auth\User as Authenticatable;
 use Illuminate\Notifications\Notifiable;
 use Laravel\Sanctum\HasApiTokens;
+use Monolog\Handler\RollbarHandler;
 
 class User extends Authenticatable
 {
     use HasApiTokens, HasFactory, Notifiable;
-
+
```
Comment (manhtd-1083) [inline app/Models/User.php:15]: thêm tab chỗ này làm chi :x

**Pull request merged**